decentraland/unity-renderer
# Fix: fixed fonts not loading on mac build

## What does this PR change?

In the mac build many fonts were not visible, this has been solved with the proper regeneration of font files

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/mac-font
2. Verify that every text is visible, no matter what platform

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
